src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -411,6 +437,13 @@ public CodeFixProvider GetSuppressionFixer(string language, ImmutableArray<Diagn
             return false;
         }
 
+        private bool IsInteractiveCodeFixProvider(CodeFixProvider provider)
+        {
+            // TODO (https://github.com/dotnet/roslyn/issues/4932): Don't restrict CodeFixes in Interactive
+            return provider is AddImport.AbstractAddImportCodeFixProvider ||
```

Comment:
For Update 1, our answer is that we don't have a story for making analyzers work in Interactive.  This is a one-off we're trying to spike for convenience.  Our goal was to provide a consistent experience (no code fixes) rather than "it works unless it doesn't".